assets/js/projects/index.js

```diff
@@ -10,121 +10,122 @@ angular.module("app", []).controller("ProjectController", function($scope) {
   $scope.repositories = [];
   $scope.activeCategory = null;
   $scope.Math = Math;
-  
+
   $scope.filterByCategory = function(value) {
-    var isInCategory = false;
-    if ($scope.activeCategory.topics) {
-      if (value.topics && value.topics.length) {
-        for(var i = 0; i < value.topics.length; i++) {
-          var valueTopic = value.topics[i];
-          if ($scope.activeCategory.topics.indexOf(valueTopic.topic.name) > -1) {
-            isInCategory = true;
-            break;
+      var isInCategory = false;
+      if ($scope.activeCategory.topics) {
+          if (value.topics && value.topics.length) {
+              for (var i = 0; i < value.topics.length; i++) {
+                  var valueTopic = value.topics[i];
+                  if ($scope.activeCategory.topics.indexOf(valueTopic.topic.name) > -1) {
+                      isInCategory = true;
+                      break;
+                  }
+              }
           }
-        }
+      } else {
+          isInCategory = true;
       }
-    } else {
-      isInCategory = true;
-    }
-    return isInCategory;
+      return isInCategory;
   };
 
   $scope.filterByPrivacy = function(value) {
-    if ($scope.showProprietaryRepositories) {
-      return true;
-    }
-    return !value.isProprietary;
+      if ($scope.showProprietaryRepositories) {
+          return true;
+      }
+      return !value.isProprietary;
   };
 
   $scope.filterByQuery = function(value) {
-    var queryMatch = true;
-    if (!$scope.query || !$scope.query.trim()) {
-      return true;
-    }
-    var query = $scope.query.toLowerCase(),
-          name = value.name.toLowerCase();
-      queryMatch = name.indexOf(query) > -1;
-    return queryMatch;
+      var queryMatch = true;
+      if (!$scope.query || !$scope.query.trim()) {
+          return true;
+      }
+      var query = $scope.query.toLowerCase(),
+          name = value.name.toLowerCase(),
+          description = value.description ? value.description.toLowerCase() : '';
+      queryMatch = name.indexOf(query) > -1 || description.indexOf(query) > -1;
```

Comment:
This is the only logic change.  Other changes are adding consistent formatting provided by [js-beautify](https://beautifier.io/)